cluster/examples/kubernetes/rook-operator.yaml

```diff
@@ -126,6 +126,9 @@ spec:
         # To disable RBAC, uncomment the following:
         # - name: RBAC_ENABLED
         #  value: "false"
+        # Rook Agent toleration. Choose between NoSchedule, PreferNoSchedule and NoExecute:
```

Comment:
we should have a doc on the operator attributes. Want to open a separate issue for that?

Reply:
Yes. Ill do that